examples/chart/teleport-cluster/values.schema.json

```diff
@@ -33,6 +33,15 @@
             "type": "string",
             "default": ""
         },
+        "proxyProtocol": {
+            "$id": "#/properties/proxyProtocol",
+            "type": "string",
+            "default": "",
+            "enum": [
+                "off",
+                "on"
+            ]
+        },
```

Comment:
Is it ok to have a single value for configuring both Auth and Proxy?
Is there a case where we might have Proxy behind a LB and Auth only available within an internal network and not behind a LB?

Reply:
Yes, but they can use the auth.teleportConfig/proxy.teleportConfig for setting a different value for each one

Reply:
> Is there a case where we might have Proxy behind a LB and Auth only available within an internal network and not behind a LB?

We had multiple feature requests and people trying to do that. I don't know if we support this currently but I think it's safe to assume some users currently expose auth directly, and maybe we'll officially support this in the future. I this case I think it's safer to set it on both. If different values are needed, the users can leverage `auth.teleportConfig` and `proxy.teleportConfig`.

Reply:
Got it, thank you :+1: